.github/workflows/lint.yml

```diff
@@ -1,6 +1,56 @@
 name: lint
 on: [pull_request]
 jobs:
+  check-opi-format:
+    runs-on: "windows-latest"
```

Comment:
does this need windows?  i couldn't see anything in https://github.com/ISISComputingGroup/ibex_gui/blob/master/base/uk.ac.stfc.isis.ibex.opis/check_opi_format.py that was hardcoded to a windows path other than in the ignored paths

Reply:
https://github.com/ISISComputingGroup/ibex_gui/blob/3e3b46a65a56075ee7ac92f6324ebc8d91fedf42/base/uk.ac.stfc.isis.ibex.opis/check_opi_info.py#L2

Reply:
Probably test could be changed if we prefer... This was just the low-cost route.